resources/qml/pages/ContactSettingsPage.qml

```diff
@@ -0,0 +1,56 @@
+import QtQuick 2.0 
+import QtQuick.Window 2.0;
+import Sailfish.Silica 1.0 
+import harbour.shmoose 1.0 
+
+Page {
+    id: page
+    allowedOrientations: Orientation.All
+    property string jid
+    property string imagePath : shmoose.rosterController.getAvatarImagePathForJid(jid)
+    property string jidName : shmoose.rosterController.getNameForJid(jid)
+
+    Column {
+
+        PageHeader {
+            title: jid+qsTr(" settings")
```

Comment:
Don't use a starting blank in the translatable string. This prevents an already existing 'settings' string to be used. If you need this, do it like:
title: jid + " " + qStr("settings")

Reply:
Please leave this open until it is resolved. Or provide a reason why it should stay like it is.